mmdet/models/losses/focal_loss.py

```diff
@@ -180,3 +233,74 @@ def forward(self,
         else:
             raise NotImplementedError
         return loss_cls
+
+
+@LOSSES.register_module()
+class FocalLossWithProb(nn.Module):
```

Comment:
Can we ommit this class and set a flag in FocalLoss. If `activated=Flase`, use the original logic, if `activated=True`, use py_focal_loss_with_prob